bigquery_schema_generator/generate_schema.py

```diff
@@ -1042,6 +1043,12 @@ def main():
         ' This can be fetched with:'
         ' `bq show --schema <project_id>:<dataset>:<table_name>',
         default=None)
+    parser.add_argument(
+        '--keep-input-sort-order',
```

Comment:
Let's use `--preserve_input_sort_order` using underscore for consistency with other flags.